Prop closer_marker
Prop farther_marker